programs/spring-util/actions/blocklog.cpp

```diff
@@ -323,27 +298,74 @@ int blocklog_actions::read_log() {
          *out << fc::json::to_pretty_string(v) << "\n";
    };
    bool contains_obj = false;
-   while((block_num <= opt->last_block) && (next = block_logger.read_block_by_num(block_num))) {
-      if(opt->as_json_array && contains_obj)
-         *out << ",";
-      print_block(next);
-      ++block_num;
-      contains_obj = true;
-   }
 
-   if(!fork_db_branch.empty()) {
-      for(auto bitr = fork_db_branch.rbegin(); bitr != fork_db_branch.rend() && block_num <= opt->last_block; ++bitr) {
-         if(opt->as_json_array && contains_obj)
+   bool print_log     = opt->print_from == print_from_t::both || opt->print_from == print_from_t::block_log;
+   bool print_fork_db = opt->print_from == print_from_t::both || opt->print_from == print_from_t::fork_db;
+
+   signed_block_ptr end;
+
+   report_time rt(print_log && print_fork_db ? "reading both log and fork_db"
```

Comment:
Maybe spell out `log` as `block log` to be clearer.

Reply:
https://github.com/AntelopeIO/spring/pull/1380/commits/4ba3c7c5b829c20646e99645aef0c280102cfbe9